1.9/overview/architecture/components.md

```diff
@@ -565,6 +576,7 @@ Just as machine operating systems need package management to install, upgrade, c
   <strong>See Also:</strong>
   <ul>
     <li><a href="https://github.com/dcos/dcos/tree/master/pkgpanda">Source</a></li>
+    <li><a href="/docs/1.9/administration/component-management/">API Reference</a></li>
```

Comment:
This is from another PR pending merge: https://github.com/dcos/dcos-docs/pull/918